admin/admin-filters-columns.php

```diff
@@ -66,7 +66,7 @@ protected function add_column( $columns, $before ) {
 		}
 
 		foreach ( $this->model->get_languages_list() as $language ) {
-			$columns[ 'language_' . $language->slug ] = $language->flag ? $language->flag . '<span class="screen-reader-text">' . esc_html( $language->name ) . '</span>' : esc_html( $language->slug );
+			$columns[ 'language_' . $language->slug ] = $language->flag ? $language->flag : '<abbr>' . esc_html( $language->slug ) . '</abbr><span class="screen-reader-text">' . esc_html( $language->name ) . '</span>';
```

Comment:
This removes the accessibilty text when displaying a flag. Could you fix?

Reply:
Hum 🤔 `alt` attribute included in the `$language->flag` is enough for accessibility and thus there is no need to duplicate this information.